integrations/standalone/tests/integration/dataclass-editor-business-dataclass.spec.ts

```diff
@@ -32,7 +34,18 @@ test('load data', async () => {
   );
 
   const row2 = table.row(2);
-  await row2.expectValues('dataClassField2', 'Date', 'DataClassField2 comment');
+  await row2.expectToHaveValues('dataClassField2', 'Date', 'DataClassField2 comment');
   await row2.click();
-  await detail.expectFieldValues('dataClassField2', 'Date', false, 'DataClassField2 comment', '');
+  await detail.expectToHaveFieldValues('dataClassField2', 'Date', false, 'DataClassField2 comment', '');
+});
+
+test('save data', async () => {
+  await editor.table.expectToHaveRowCount(3);
+
+  await editor.addField('newAttribute', 'String');
+
+  const editor2 = await DataClassEditor.openEngine(editor.page, file);
+
+  await editor2.table.expectToHaveRowCount(4);
+  await editor2.table.row(3).expectToHaveValues('newAttribute', 'String', '');
 });
```

Comment:
you now edit data on a real engine file. 
So if this test runs first then the other ones will fail!

For change file test we should work with create a temp data class first via API.